libsolidity/ast/Types.cpp

```diff
@@ -1661,6 +1662,37 @@ MemberList::MemberMap ContractType::nativeMembers(ContractDefinition const*) con
 				&it.second->declaration()
 			));
 	}
+	// Add overloads from address only if there is no conflict
+	for (auto it = addressMembers.begin(); it != addressMembers.end(); ++it)
+	{
+		bool clash = false;
+		for (auto const member: members)
```

Comment:
Can you extract this into a function?

Reply:
Or perhaps everything startign from `add overloads from address....`?

Reply:
I was thinking about the same but not sure it helps much. It is quite specific.

Reply:
Done.